src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -84,12 +84,12 @@ impl Command {
                 Ok(0) => return Ok((p, ours)),
                 Ok(8) => {
                     let (errno, footer) = bytes.split_at(4);
-                    assert!(
-                        combine(CLOEXEC_MSG_FOOTER) == combine(footer.try_into().unwrap()),
+                    assert_eq!(
+                        CLOEXEC_MSG_FOOTER, footer,
```

Comment:
I think rustfmt will dislike that. `footer,` should be in a newline.

Reply:
I thought so. But rustfmt wants the current change format.

Reply:
Wow... that is *very* unexpected to me. Interesting.